_i18n/es.yml

```diff
@@ -360,17 +360,16 @@ projects:
-                   <p>In an effort to modernise and improve their IT capabilities, ASOS underwent a programme to improve the speed of change, the ease, and rate of delivery and a general upskilling of staff in this area. They realised they were missing some key skills, particularly around agile technical practices, and asked us to help.</p>
-    our_sc_approach: <p>We helped by rapidly building and re-architecting some of their core systems, adding in levels of redundancy and caching that allowed them to cope with peak traffic. We also built fallover contingency libraries to implement cross-region disaster recovery and implemented cross-region high availability clusters.</p>
-                     <p>The work was in NET ASP NET MVC with SQL Azure databases, hosted in Azure The site also employed some open source caching technologies such as Redis, to improve availability. </p>
-                     <p>Whilst their existing development teams were already fairly established, particularly around Agile processes, we identified the need for these teams to be upskilled technical Agile practices We implemented a large-scale Software Craftsmanship coaching program across their 40+ development teams, introducing or reinforcing key skills around <a href="/services">Extreme Programming (XP)</a>, TDD, SOLID principles, <a href="/services">Continuous Integration</a>, <a href="/services">Continuous Delivery</a> and a <a href="/services">culture of DevOps</a> </p>
-    outcome: <p>ASOS has made significant steps along the road to Agile software development, adopting a host of new techniques rapidly Iterations are more organized and ceremonies are delivering more value.</p>
-             <p>We changed the technical practices within the team, raising the overall quality of code. Several team members that went through the training courses have been promoted into key engineering leadership positions.</p>
-             <p>ASOS continues to thrive and improve and we are proud of the help we provided while there We believe that ASOS's attitude and desire for growth and change aligned well with Codurance's expertise, coaching, and support, resulting in many enriching experiences for ASOS's staff.</p>
+    subtitle: <p>El despliegue manual de muchas etapas, que requería multiples equipos, tenía un impcto negativo en la habilidad de la organización para desplegar cambios regularmente. La metodología de de Software Craftmanship fue vista como la manera de mejorar la calidad del código y proveer una plataforma hacia Continous Delivery</p>
+    about: ASOS es un negocio de moda online, vende moda de vanguardia y apoya una prospera comunidad de moda. Comenzó como un blog de moda, la marca ahora ofrece a sus clientes aplicaciones móviles y una revista además de su página web. El último año ASOS registró 13,4 millones de usuarios activos.
```

Comment:
Missing tilde on pr**o**spera
I'd say `Comenzando como un blog de moda` - Does anyone else have any thoughts about this?